### PR TITLE
After test

### PR DESCRIPTION
**Pull request by Dorosh90**
- [DEMO LINK](https://Dorosh90.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Dorosh90.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

**Review by sTorba24 — CHANGES_REQUESTED** [review]
add links to description please

**Comment by Dorosh90** [comment]
file:///Users/dorosh/project3/layout_search-bar-airbnb/backstop_data/html_report/index.html
https://dorosh90.github.io/layout_search-bar-airbnb/

**Comment by Dorosh90** [comment]
https://dorosh90.github.io/layout_search-bar-airbnb/

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  line-height: 22px;
+  font-size: 16px;
+}
+
+.big__input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.big__input:focus {
+  font-family: Avenir-Heavy, sans-serif;
```
Comment (OleziO) [inline src/style.css:55]: Add `outline: 0` to prevent border when on focus. Check other cases.

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  width: 100%;
+  height: 42px;
+  padding-left: 33px;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  line-height: 19.12px;
+  font-size: 14px;
+  font-weight: 400;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
```
Comment (OleziO) [inline src/style.css:91]: Use HEX instead of RGBA. Check other similar cases.

**Review by OleziO — APPROVED** [review]

**Pull request closed without merging**